python/cudf/cudf/_lib/pylibcudf/libcudf/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # the License.
 # =============================================================================
 
-set(cython_sources aggregation.pyx binaryop.pyx copying.pyx replace.pyx reduce.pxd round.pyx
+set(cython_sources aggregation.pyx binaryop.pyx copying.pyx reduce.pyx replace.pyx round.pyx
```

Comment:
Is this a necessary change for this PR, or just a chore?

Reply:
For reasons, I think I needed to change `reduce.pxd` to `reduce.pyx`, and took the opportunity to alphabetise as well.